src/ssp.cpp

```diff
@@ -138,81 +54,180 @@ void InitcCubic(SSP_INIT_ARGS)
         ssp->NPts, iBCBeg, iBCEnd, ssp->NPts);
 }
 
-void InitQuad(SSP_INIT_ARGS)
+void ReadQuad(PrintFileEmu& PRTFile, SSPStructure* ssp, std::string FileRoot)
 {
-    real Depth = x[1];
-    ReadSSP(CALL_READ_SSP_ARGS);
-    
     // Read the 2D SSP matrix
     PRTFile << "__________________________________________________________________________\n\n";
     PRTFile << "Using range-dependent sound speed\n";
-    
+
     LDIFile SSPFile(FileRoot + ".ssp");
     LIST(SSPFile); SSPFile.Read(ssp->Nr);
     PRTFile << "Number of SSP ranges = " << ssp->Nr << "\n";
-    
-    if(ssp->Nr < 2){
+
+    if (ssp->Nr < 2) {
         PRTFile << "READIN: Quad: You must have a least two profiles in your 2D SSP field\n";
         std::abort();
     }
-    
-    checkallocate(ssp->cMat ,  ssp->NPts    * ssp->Nr);
-    checkallocate(ssp->czMat, (ssp->NPts-1) * ssp->Nr);
+
+    checkallocate(ssp->cMat, ssp->NPts * ssp->Nr);
```

Comment:
Let's try to avoid commits changing whitespace in lines which aren't actually having content changed. I'd be open to choosing some auto-format tool for the repo if you'd like, but that'd be a future commit.

Reply:
Yes, whitespace changed in the functions I moved around. I reverted where I could.
A linter would be great. To discuss elsewhere.